<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>20</y>
      <width>661</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Navigation</string>
    </property>
    <widget class="QPushButton" name="pushButtonBegin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>|&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNext">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPrevious">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;|</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxHashTag">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>HashTag</string>
    </property>
    <widget class="QComboBox" name="comboBoxHashTag">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSelect_image">
    <property name="title">
     <string>Select image</string>
    </property>
    <addaction name="actionSelectImageBegin"/>
    <addaction name="actionSelectImageNext"/>
    <addaction name="actionSelectImagePrevious"/>
    <addaction name="actionSelectImageEnd"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelect_image"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSelectImageBegin">
   <property name="text">
    <string>Begin</string>
   </property>
  </action>
  <action name="actionSelectImageNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionSelectImagePrevious">
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionSelectImageEnd">
   <property name="text">
    <string>End</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
